src/components/dialogs/parameters/sensi/table-row.tsx

```diff
@@ -51,14 +49,10 @@ const TableRowComponent: FunctionComponent<TableRowComponentProps> = ({
                 >
                     <span hidden={!hover}>
                         <IconButton
-                            color="primary"
+                            sx={{ border: 'solid 0px rgba(0,0,0,0.26)' }}
```

Comment:
maybe we can create a new field in global theme for the color of the border

Reply:
done